scripts/check_material_profiles.py

```diff
@@ -23,7 +24,7 @@ def _get_guid(self, content: str) -> str:
                 break
         return guid
 
-    def get_materials_dir(self, dirpath: str):
+    def get_materials_dir(self, dirpath: str) -> str:
```

Comment:
I understand that you want to bring this utility script under code style, but it's missing multiple of our [code style conventions](https://github.com/Ultimaker/Meta/blob/master/general/generic_code_conventions.md#naming-conventions) that are not being tested by Mypy now.

In general though, for these utility scripts, I don't think it's really worth our time. It's not like it's undergoing a lot of maintenance.